lite2/verifier.go

```diff
@@ -174,9 +174,10 @@ func verifyNewHeaderAndVals(
 	}
 
 	if !bytes.Equal(untrustedHeader.ValidatorsHash, untrustedVals.Hash()) {
-		return errors.Errorf("expected new header validators (%X) to match those that were supplied (%X)",
+		return errors.Errorf("expected new header validators (%X) to match those that were supplied (%X) at height %d",
```

Comment:
it's the responsibility of the caller to add this info since a) it can be obtained outside this function b) function itself does not change the value of `height` param

Reply:
What do you mean by this - do you thing adding at heigh %d is unnecessary

Reply:
> do you thing adding at heigh %d is unnecessary

yes